front/src/assets/variables.tsx

```diff
@@ -0,0 +1,45 @@
+type Media = {
+  xs: string;
+  sm: string;
+  md: string;
+  lg: string;
+  xl: string;
+  xxl: string;
+};
+
+export const media: Media = {
```

Comment:
onst mediaQ = (size: number): string =>
  `@media only screen and (max-width: ${size}px)`

export const media = {
  xs: mediaQ(380),
  sm: mediaQ(576),
  md: mediaQ(768),
  lg: mediaQ(992),
  xl: mediaQ(1200),
  xxl: mediaQ(1440),
}

typow tu nie ma sensu robić

Reply:
Done.